Header header
uint16[]  id # landmark id
float64[] range # x coordinates of the cylinders' centers
float64[] bearing # y coordinates of the cylinders' centers
float64[] radius # cylinders' radiuses
